articles/nodejs-structural-comparisons/index.md

```diff
@@ -0,0 +1,562 @@
+### Node.js Structural Comparisons
+
+Node.js is a type of computer language based on JavaScript. When structuring codes to produce and generate results, the number of lines of code can become time-efficient and eventually valued. This tutorial will display numerous scenarios' structural breakdowns by comparing Node.js to R-Programming, C/C++, and Java.  
+
+R-Programming, C/C++, and Java were selected to differentiate components that one can find quickly with Node.js. The tutorial will extend to other programming structures such as sequencing, organization, features, and available options. Applying these concepts might benefit specific situations such as data science, applications with possible graphic user interface preferences, and general scripts.  
+
+#### Table of Contents
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Modules and Libraries](#modules-and-libraries)
+* [Variables with Value](#variables-with-value)
+* [Graphing](#graphing)
+* [Graphical User Interface Applications](#graphical-user-interface-applications)
+
+#### Prerequisites
+
+* Windows device.
+* Previous knowledge of installing modules and libraries.
+* R-Programming.
+* Kali Linux (to compile C and C++ codes).
+* NetBeans (to build in Java).
+* Node.js application (to write in Node.js).
+* Plotly account (to graph).
+* Some coding knowledge or experience.
+
+Once most of the prerequisites are met, let's begin.  
+
+#### Modules and Libraries
+
+##### Node.js
+
+Node.js computer language is written in a unique structure. One characteristic to consider is the steps to create or add modules into Node.js. As each coding language includes its sequenced procedure, Node.js also involves flexible options.  
+
+We can find Node.js modules on open-source websites. In Node.js, folders and JavaScript files are known to hold modules. Modules inside the `node_module` folder can be used in the Node.js application.  
+
+A branched-out path directory looks similar to the picture below.  
+
+[directory](/engineering-education/nodejs-structural-comparisons/path.jpg)  
```

Comment:
In order to display images, you need to add an exclamatory mark before the markdown command mentioned above. Currently the image is not displayed. Rectify this for all images.

Reply:
Thank you. Will do.